dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerMainIntegrationTest.java

```diff
@@ -92,27 +92,29 @@ public void testArtifacts()
       throws IOException, RepositoryException, TransformerException, XMLStreamException {
     try {
       LinkageCheckerMain.main(
-          // protobuf-java:3.13.0 in
-          new String[] {"-a", "com.google.cloud:google-cloud-firestore:1.35.2"});
+          // google-http-client-appengine depends on appengien-api-1.0-sdk which shades its
+          // dependencies, resulting in many linkage errors.
+          new String[] {"-a", "com.google.http-client:google-http-client-appengine:1.39.2"});
       fail("LinkageCheckerMain should throw LinkageCheckResultException upon errors");
     } catch (LinkageCheckResultException expected) {
```

Comment:
Would `assertThrows` work here?

Reply:
(I didn't know the feature of JUnit 5. Good to know. Thanks)
This project uses JUnit 4, which doesn't have the method.